.docs/user-guide/storage-providers.md

```diff
@@ -675,22 +706,58 @@ The DigitalOcean block storage driver has the following requirements:
 * Valid DigitalOcean [access token](https://goo.gl/iKoAec)
 
 #### Configuration
-The following example illustrates a minimal configuration for the DigitalOcean
+The following is an example with all possible fields configured. For a running
+example see the `Examples` section.
```

Comment:
Hi @codenrhoden,

> see the `Examples` section.

I always like to add a _please_ in front of a directive. Just a nice thing to do :) Also, if you're asking someone to look at something why not link to it with an anchor? It's also possible to add custom anchors to avoid using the ones generated by mkdocs. Here's an example:

```
<a class="headerlink hiddenanchor" name="aws-ebs"></a>
```